theme/ItaliaTheme/Views/_notizia.scss

```diff
@@ -0,0 +1,23 @@
+.contenttype-news-item {
```

Comment:
occhio che la gallery nn c'è solo nelle notizie ma anche in altri ct. 
Guarda dove è usato il componente che disegna la gallery per capire in quali ct viene usato. 
Mettiamo il fix per tutti i ct